pivy-importer/build.gradle

```diff
@@ -23,6 +23,12 @@ dependencies {
     compile 'ch.qos.logback:logback-classic:1.1.3'
     compile 'org.apache.commons:commons-compress:1.10'
     compile 'commons-io:commons-io:2.4'
+
+    testCompile group: "junit", name: "junit", version: "4.+"
+    testCompile group: 'org.mockito', name: 'mockito-core', version: '2.+'
+    testCompile ('org.spockframework:spock-core:1.1-groovy-2.4') {
+        exclude module: 'groovy-all' // Avoid conflict with version of Groovy loaded by Gradle
```

Comment:
I don't think you need the exclude since we're not including `gadleApi()` or `groovyLocal()` here.

Reply:
Good point, removed!